code/__DEFINES/atmospherics.dm

```diff
@@ -70,12 +70,26 @@
 #define MIN_TOXIC_GAS_DAMAGE				1
 #define MAX_TOXIC_GAS_DAMAGE				10
 #define MOLES_PLASMA_VISIBLE				0.5		//Moles in a standard cell after which plasma is visible
+//General assmos defines.
+#define WATER_VAPOR_FREEZE					200
+#define BROWNS_FORMATION_ENERGY				100000
+#define GAS_STIM_MINIMUM					0.002
+#define TRITIUM_BURN_OXY_FACTOR				2
+#define SUPER_SATURATION_THRESHOLD			96
+#define STIMULUM_HEAT_SCALE					100000
+#define STIMULUM_FIRST_RISE					0.75
+#define STIMULUM_FIRST_DROP					0.05
+#define STIMULUM_SECOND_RISE				0.001
+#define STIMULUM_ABSOLUTE_DROP				0.0000062
+#define REACTION_OPPRESSION_THRESHOLD		5

```

Comment:
still think reaction defines could be better off moved to reactions.dm. worth considering for later

Reply:
Yes, it's one movement, and you can undef them at the end anyway, do this please.